### PR TITLE
Fix obc ui test leftovers

### PR DESCRIPTION
**Pull request by ebondare**

### CODE REVIEW

**Comment by ebondare** [comment]
https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/29909/console successful test run

**Review by DanielOsypenko — DISMISSED** [review]

**Review by OdedViner — APPROVED** [review]

File: tests/ui/test_non_admin_user.py [hunk 1]
```diff
@@ -52,22 +51,18 @@ def finalizer():
...
-            f"--user={user[0]} "
-        )
+    def test_project_admin_obcs_access(self, user_factory, login_factory):
+        """
+        Test if project admin can view the list of OBCs
 
-        """Login using created user"""
+        """
+        user = user_factory()
+        sleep(5)
```
Comment (am-agrawa) [inline tests/ui/test_non_admin_user.py:60]: It would be good to use explict wait types in UI automation..

File: tests/ui/test_non_admin_user.py [hunk 2]
```diff
@@ -52,22 +51,18 @@ def finalizer():
...
-
-        """ Create RoleBinding """
-        ocp_obj = ocp.OCP()
-        ocp_obj.exec_oc_cmd(
-            f"-n openshift-storage create rolebinding {user[0]} --role=mcg-operator.v4.11.0-noobaa-odf-ui-548459769c "
-            f"--user={user[0]} "
-        )
+    def test_project_admin_obcs_access(self, user_factory, login_factory):
+        """
+        Test if project admin can view the list of OBCs
```
Comment (am-agrawa) [inline tests/ui/test_non_admin_user.py:56]: I don't understand the meaning of project admin.

File: ocs_ci/ocs/ui/page_objects/object_bucket_claims_tab.py [hunk 3]
```diff
@@ -62,30 +62,41 @@ def __init__(self):
         }
         self.sc_loc = self.obc_loc
 
-    def check_obc_option(self, text="Object Bucket Claims"):
-        """check OBC is visible to user after giving admin access"""
+    def check_obc_option(self, username, text="Object Bucket Claims"):
+        """
+        check OBC is visible to user after giving admin access
```
Comment (am-agrawa) [inline ocs_ci/ocs/ui/page_objects/object_bucket_claims_tab.py:67]:
```suggestion
        Check OBC is visible to user after giving admin access
```

File: tests/ui/test_non_admin_user.py (hunk 2 above)
Comment (ebondare) [reply tests/ui/test_non_admin_user.py:56]: It's not kubeadmin but the user that has admin access to a specific project. I'll try to think of a way how to describe it better

**Comment by ebondare** [comment]
https://ocs4-jenkins-csb-odf-qe.apps.ocp-c1.prod.psi.redhat.com/job/qe-deploy-ocs-cluster/29920 new successful test run

File: tests/ui/test_non_admin_user.py (hunk 1 above)
Comment (ebondare) [reply tests/ui/test_non_admin_user.py]: Thank you, fixed

File: tests/ui/test_non_admin_user.py (hunk 2 above)
Comment (ebondare) [reply tests/ui/test_non_admin_user.py]: Fixed

File: ocs_ci/ocs/ui/page_objects/object_bucket_claims_tab.py (hunk 3 above)
Comment (ebondare) [reply ocs_ci/ocs/ui/page_objects/object_bucket_claims_tab.py]: Fixed

**Review by am-agrawa — APPROVED** [review]

**Review by OdedViner — APPROVED** [review]

**Review by DanielOsypenko — APPROVED** [review]

**Review by vavuthu — APPROVED** [review]

**Comment by ebondare** [comment]
Fixes https://github.com/red-hat-storage/ocs-ci/issues/8540

**Pull request merged**